query.go

```diff
@@ -111,3 +186,53 @@ func (q Query) IsEmpty() bool {
 	}
 	return true
 }
+
+// RemoveWithout returns query removed --without key
+func (q Query) RemoveWithout() Query {
+	removed := Query{}
+	for k, v := range q {
+		if strings.HasPrefix(k, "without") {
+			removed[k] = ""
+		} else {
+			removed[k] = v
+		}
+	}
+	return removed
+}
+
+// Valid returns true if query isn't conflicted
+func (q Query) Valid() bool {
+	hasWithoutSerial := q["without-serial"]
+	if hasSerial := q["serial"]; len(hasSerial) > 0 && len(hasWithoutSerial) > 0 {
+		return false
```

Comment:
Can't users find out why it failed?

Reply:
Sabakan returns `"'with' and 'without' options about the same things are specified."` when users specifies bothe with and without.